### DISCUSSION TITLE
clarification requested for Wiki entry

### DISCUSSION THREAD

**Discussion by River-Mochi** [Ralim/IronOS#1375] [General]
This sentence is in Pinecil  Wiki

not sure what is the intended meaning of last part of this:

"This enables some QC adapters to work (i.e., some Baseus QC chargers), but may result in problems with PD negotiation on slower PD adapters.  For certain adapters, a PD timeout value of 15s is needed **while most of PD adapters are also work using this value.**"

**Comment by Ralim** [comment]
-> "Most PD adaptors can work fine with a value of 15 but it increases the probability of issues with QC chargers"

As this setting adjusts the time between giving up on PD and starting QC, if your QC charger needs you to start negotiation quickly you want a smaller value. Meanwhile, PD negotiation speed depends on the adaptor. Generally PD adapters are really fast, but some are slow.
